extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeChannelExtractorTest.java

```diff
@@ -648,4 +649,102 @@ public void testVerified() throws Exception {
             assertFalse(extractor.isVerified());
         }
     }
+
+    /**
+     * Test the extraction of channel data from a RichGridRenderer (currently A/B tested, as of 11.10.2022)
+     */
+    public static class RichGrid implements BaseChannelExtractorTest {
```

Comment:
I am not sure if a test with a new channel is needed, as the VSauce channel has the three tabs:

<img src="https://user-images.githubusercontent.com/74829229/198875516-b4cd9a99-dc96-47b8-8920-d07951fd8eb1.png" width="600px" alt="VSauce_channel_new_channel_tabs">